scripts/legacy/sample_additional_info.py

```diff
@@ -174,7 +171,39 @@ def get_url_formatters_for_properties(property_mapping_path, output):
         query = (
             """SELECT * WHERE { wd:%s wdt:P1630 ?formatterUrl . }""" % prop_id
         )
-        for r in make_request(query):
+        for r in _make_request(query):
             formatters[prop_id] = r['?formatterUrl']
 
     json.dump(formatters, open(filepath, 'w'), indent=2, ensure_ascii=False)
+
+
+def query_info_for(qids_bucket, properties):
+    """Given a list of wikidata entities returns a query for getting some external ids"""
+
+    query = """SELECT * WHERE{ VALUES ?id { %s } """ % ' '.join(qids_bucket)
+    for i in properties:
+        query += """OPTIONAL { ?id wdt:%s ?%s . } """ % (i, i)
+    query += """}"""
+    return query
+
+
+def query_wikipedia_articles_for(qids_bucket):
+    """Given a list of wikidata entities returns a query for getting wikidata articles"""
+
+    query = """SELECT * WHERE{ VALUES ?id { %s } """ % ' '.join(qids_bucket)
+    query += """OPTIONAL { ?article schema:about ?id . }"""
+    query += """}"""
+    return query
+
+
+def query_birth_death(qids_bucket):
```

Comment:
This public function does not have typing annotations

Reply:
Same comment as above.